### ISSUE TITLE
PubSub does not work under Context API

### ISSUE THREAD

**Issue by bmalbusca** [mroderick/PubSubJS#195]
The comm between components stoped after added Context API to the project, which provides contexts to the  parent nodes of those components that use PubSub.

**Comment by mroderick** [comment]
What is "Context API"?

**Comment by bmalbusca** [comment]
Context API is a built-in library from ReactJS that makes possible to share an object/variables, in a form of contexts, directly to other components from the components tree. The oficial docs: https://reactjs.org/docs/context.html

**Comment by mroderick** [comment]
Would it be possible for you to create a reduced example that shows how this is working without Context and how it's not working WITH Context? Perhaps then some React experts might see this and be able to help with this.

I don't work with React.

**Comment by bmalbusca** [comment]
Yes, I will post here a simpler example. Thank you @mroderick for your help.
